src/main/jbake/content/blog/2017/06/26/opentracing-appmetrics.adoc

```diff
@@ -189,7 +189,7 @@ minikube service prometheus --url
 kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-kubernetes/master/all-in-one/jaeger-all-in-one-template.yml
```

Comment:
You can use tagged version(not master), wich will always work with this blog. In the future there might be more changes which will break this.